src/manifest.rs

```diff
@@ -293,10 +293,15 @@ impl LocalManifest {
         Ok(LocalManifest { manifest, path })
     }
 
+    /// Check if local manifest is virtual, i. e. corresponds to workspace root
+    pub fn is_virtual(&self) -> bool {
+        !self.data["workspace"].is_none()
```

Comment:
The existence of a `workspace` table does not make it virtual.  To be virtual, it also needs to not have a package.

Check out https://github.com/crate-ci/clap-cargo/blob/master/src/workspace.rs#L29 for a good approximation of what `cargo` does.

Reply:
OK, I'll fix it a bit later.

Reply:
Alternatively, we could just port the project over to `clap-cargo`.

Reply:
`clap-cargo` look like an amazing project! I'm willing to help with the porting.